frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -324,7 +324,7 @@ def comparison(
                     if operand.bit_width != operand.original_bit_width:  # pragma: no cover
                         highlights[operand.origin].append(
                             "("
-                            f"note that it's assigned {operand.bit_width}-bits "
+                            f"note that it's assigned to {operand.bit_width}-bits "
```

Comment:
"assign something to someone"
doesn't the "to" refer to "it" and not the bitwidth ?

Reply:
actually you are right my suggestion was no write as well, I agree with the  "assign something to someone" form but I still think there's a small issue with the initial sentence, let me change it again !